### PR TITLE
add nsfw for vc

### PR DESCRIPTION
**Pull request by RealYusufIsmail**
closes : #1126 
Has been tested

This pr adds the ability to check if a vc is nsfw
You can also now create a vc and set it as nsfw

### CODE REVIEW

**Comment by RealYusufIsmail** [comment]
@KILLEliteMaste can you please check this pr

**Comment by Zeitheron** [comment]
I would also add the `updateNsfwFlag` method since it is present in [ServerTextChannel#L79](https://github.com/Javacord/Javacord/blob/master/javacord-api/src/main/java/org/javacord/api/entity/channel/ServerTextChannel.java#L79)

File: javacord-core/src/main/java/org/javacord/core/entity/channel/ServerVoiceChannelUpdaterDelegateImpl.java [hunk 1]
```diff
@@ -31,6 +31,11 @@ public class ServerVoiceChannelUpdaterDelegateImpl extends RegularServerChannelU
      */
     protected boolean modifyCategory = false;
 
+    /**
+     * Whether the vc in nsfw or not.
```
Comment (Saladoc) [inline javacord-core/src/main/java/org/javacord/core/entity/channel/ServerVoiceChannelUpdaterDelegateImpl.java:35]:
```suggestion
     * Whether the vc is nsfw or not.
```

**Review by Saladoc — APPROVED** [review]
One typo, but apart from that LGTM

**Review by Saladoc — CHANGES_REQUESTED** [review]
As per the conversation on discord, please test the changes and verify the correctness of your code.

**Comment by RealYusufIsmail** [comment]
Will get on this pr tomorrow

**Comment by RealYusufIsmail** [comment]
nsfw can be null for vc

**Review by Saladoc — CHANGES_REQUESTED** [review]
Code-wise everything appears to be in order. 
Please 
 - Rebase and squash your commits
 - Edit the Pull Request description to include at least the changelog portion from the current issue template.

**Review by KILLEliteMaste — CHANGES_REQUESTED** [review]

**Review by Saladoc — CHANGES_REQUESTED** [review]

**Review by KILLEliteMaste — APPROVED** [review]

**Review by Saladoc — APPROVED** [review]

**Pull request merged**